src/app/(auth)/sign-in/page.tsx

```diff
@@ -41,35 +45,59 @@ function SingIn() {
   };
 
   return (
-    <div>
-      <Formik
-        initialValues={iniatialValues}
-        onSubmit={(values) => signInUser(values)}
+    <div className="grid grid-cols-1 md:grid-cols-2 h-screen">
+      <div
+        className="hidden lg:block bg-cover bg-center"
+        style={{ backgroundImage: "url('/images/login-bg.png')" }}
```

Comment:
unless it is dynamic, we will not use style please use tailwind background class

Reply:
you can use`bg-[url]` in this case